File: src/main/java/com/indiepost/dto/stat/TimeDomainStat.java
package com.indiepost.dto.stat;

import java.time.LocalDateTime;

/**
 * Created by jake on 17. 4. 24.
 */
public class TimeDomainStat {

    private LocalDateTime statDateTime;

    private Long statValue;

    public TimeDomainStat() {
    }

    public TimeDomainStat(LocalDateTime statDateTime, Long statValue) {
        this.statDateTime = statDateTime;
        this.statValue = statValue;
    }

    public LocalDateTime getStatDateTime() {
        return statDateTime;
    }

    public void setStatDateTime(LocalDateTime statDateTime) {
        this.statDateTime = statDateTime;
    }

    public Long getStatValue() {
        return statValue;
    }

    public void setStatValue(Long statValue) {
        this.statValue = statValue;
    }
}

File: src/main/kotlin/com/indiepost/dto/analytics/TimeDomainStat.kt
package com.indiepost.dto.analytics

import java.time.LocalDateTime

/**
 * Created by jake on 17. 4. 24.
 */
class TimeDomainStat {

    var statDateTime: LocalDateTime? = null

    var statValue: Long? = null

    constructor() {}

    constructor(statDateTime: LocalDateTime, statValue: Long?) {
        this.statDateTime = statDateTime
        this.statValue = statValue
    }
}
